qa-test-apps/ignore-missing-resources/with-diff/.helmignore

```diff
@@ -0,0 +1,23 @@
+# Patterns to ignore when building packages.
```

Comment:
nit: with only YAML files in this directory, this file may not be necessary.
Same remark about `without-diff`.

Reply:
Removed `.helmignore` file thanks for the suggestion.